guidelines/mig-drd.md

```diff
@@ -420,6 +425,14 @@ MONAI Deploy Informatics Gateway R1
 Since MIG is the integration point with the medical systems for MONAI Deploy, it shall notify other subsystems when data is received and ready.
 
 
+#### Verification Strategy
+
+Send a dataset to MIG and expect MIG to emit a message to notify other subsystems of the location of the data.
+
+#### Target Release
+
+MONAI Deploy Informatics Gateway R1
```

Comment:
Why is R1 here and 0.2.0 above? Shouldn't it be the same?

Reply:
This is for a `REQ-FNC-02` where I forgot to include the target release and test strategy.